lib/widget/camera_widget.dart

```diff
@@ -39,30 +40,36 @@ class Camera extends StatefulWidget
     return {
       'height': (value) => _controller.height = Utils.optionalDouble(value),
       'width': (value) => _controller.width = Utils.optionalDouble(value),
+      'ImagePreviewheight': (value) =>
+          _controller.ImagePreviewheight = Utils.optionalDouble(value),
+      'ImagePreviewwidth': (value) =>
+          _controller.ImagePreviewwidth = Utils.optionalDouble(value),
+      'fullImageWidth': (value) =>
```

Comment:
Why is this needed? Just size to the camera's width/height.

Reply:
this is basically full image view when user select image form gallery or capture image form camera. if user can change preview image height